### PR TITLE
data: aakashditya

### PR DESCRIPTION
**Pull request by aakashditya**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5176"></a>

### CODE REVIEW

**Review by TRohit20 — COMMENTED** [review]
Everything seems to be in order. LGTM!

File: data/aakashditya.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Aakash Ditya",
+  "type": "personal",
+  "bio": "Currently learning & building in Full-stack development",
+  "displayStatsPublic" : true,
+  "avatar": "https://github.com/aakashditya.png",
```
Comment (swagataroy30) [inline data/aakashditya.json:6]:
```suggestion
```

File: data/aakashditya.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Aakash Ditya",
+  "type": "personal",
+  "bio": "Currently learning & building in Full-stack development",
+  "displayStatsPublic" : true,
```
Comment (swagataroy30) [inline data/aakashditya.json:5]:
```suggestion
```

File: data/aakashditya.json (hunk 1 above)
Comment (swagataroy30) [inline data/aakashditya.json:6]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture and also `"displayStatsPublic" : true` is not required anymore. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

File: data/aakashditya.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Dityaaakash",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on Github",
+      "url": "https://github.com/aakashditya",
+      "icon": ""
+    },
```
Comment (swagataroy30) [inline data/aakashditya.json:17]:
```suggestion
    }
```

**Pull request merged**